src/beautifiers/prettydiff.coffee

```diff
@@ -117,6 +117,7 @@ module.exports = class PrettyDiff extends Beautifier
         source: text
         lang: lang
         mode: "beautify"
+        crlf: getDefaultLineEnding() # solved issue 1457
```

Comment:
You will need to add `eol` option to the languages which should support this option, such as https://github.com/Glavin001/atom-beautify/blob/master/src/languages/javascript.coffee#L33